public/app/plugins/datasource/graphite/datasource.test.ts

```diff
@@ -197,8 +202,8 @@ describe('graphiteDatasource', () => {
       };
 
       beforeEach(async () => {
-        datasourceRequestMock.mockImplementation((options: any) => {
-          return Promise.resolve(response);
+        fetchMock.mockImplementation((options: any) => {
+          return createFetchResponse(response);
```

Comment:
nit: I see a lot of mockImplementation() but I'm missing a mockClear() somewhere. Maybe I need to look harder.

Reply:
You have a `jest.clearAllMocks();` in the top `beforeEach` that should cover this.

Reply:
Oops, I missed that 🙈